Foundation/NSPathUtilities.swift

```diff
@@ -574,6 +574,7 @@ extension FileManager {
 }
 
 public func NSSearchPathForDirectoriesInDomains(_ directory: FileManager.SearchPathDirectory, _ domainMask: FileManager.SearchPathDomainMask, _ expandTilde: Bool) -> [String] {
+    #if os(iOS)
```

Comment:
We run on Mac for testing purposes but never on iOS.

Anyway, I'm pretty sure that this does not return a location inside the main bundle even on iOS. It returns something inside the container, which is not the same as the main bundle.